Makefile

```diff
@@ -161,9 +183,19 @@ docker-push:
 release:
 	@sed -i -e 's/^VERSION := .*/VERSION := ${NEWVERSION}/' ./Makefile
 
-release-manifest:
+release-manifest: generate-helm-chart
 	@sed -i'' -e 's@image: $(REPOSITORY):.*@image: $(REPOSITORY):'"$(NEWVERSION)"'@' ./config/manager/manager.yaml ./deploy/gatekeeper.yaml
 
+generate-helm-chart:
```

Comment:
This name is currently misleading. I suggest you move the helm-specific logic in `manifests` here and make `manifests` a dependency of `generate-helm-chart`.

Reply:
Since this one (which updates the charts, not the templates) requires a version to be supplied I thought it would be better to keep it separate from the template generation. If we put it all together then `make manifests` would now require a version to be passed in. (Unless I'm misunderstanding what you are suggesting.)

Reply:
We should move the version-updating stuff to `release-manifest`

I think you're right that there's a problem with dependencies in my original suggestion. I think we can just remove this rule altogether once `release-manifest` is updated

Reply:
Sure thing, I've combined it all and removed the extra rule.